benchmark/benchmark_block_adjacent_difference.cpp

```diff
@@ -338,12 +338,19 @@ auto run_benchmark(benchmark::State& state, hipStream_t stream, size_t N)
     HIP_CHECK(hipFree(d_output));
 }
 
-#define CREATE_BENCHMARK(T, BS, IPT, WITH_TILE) \
-benchmark::RegisterBenchmark( \
-    (std::string("block_adjacent_difference<" #T ", " #BS ">.") + name + ("<" #IPT ", " #WITH_TILE ">")).c_str(), \
-    &run_benchmark<Benchmark, T, BS, IPT, WITH_TILE>, \
-    stream, size \
-)
+# define CREATE_BENCHMARK(T, BS, IPT, WITH_TILE)                                \
+    benchmark::RegisterBenchmark(                                               \
+        (std::string("block_adjacent_difference<Datatype:" #T                   \
+            ",Block Size:" #BS                                                  \
+            ">.SubAlgorithm Name:") + name                                      \
+            + std::string("<Items Per Thread:" #IPT                             \
+            ",With Tile:" #WITH_TILE ">")                                       \
+        ).c_str(),                                                              \
+        &run_benchmark<Benchmark, T, BS, IPT, WITH_TILE>,                       \
+        stream,                                                                 \
+        size                                                                    \
+    )                                                                           
```

Comment:
Was there a reason for this change? There also might be some internal benchmarking scripts that may need to be updated.

Reply:
Hi @Naraenda, we do have an internal tool for performance tracking (Performance Tracking System - PTS) and for that we need the information regarding “Field” and “Value” for the performance results obtained in the benchmarks (in .csv or .json format). On top of hipCUB, we have similar requirements for rocPRIM, rocRAND, and rocThrust. I have also identified some updates that need to be done for rocPRIM and rocRAND. I will communicate with you in advance about those updates. In the case of hipCUB, please let us know what else needs to be updated. Thanks.